jme3-examples/src/main/java/jme3test/texture/TestSkyRotation.java

```diff
@@ -128,17 +128,17 @@ public void onAction(String name, boolean ongoing, float ignored) {
          */
         if (name.equals("left")) {
             angle += 0.1f; // radians
-            System.out.print("rotate floor and sky leftward ...");
+            System.out.println("rotate floor and sky leftward ...");
         } else if (name.equals("right")) {
             angle -= 0.1f; // radians
-            System.out.print("rotate floor and sky spatials rightward ...");
+            System.out.println("rotate floor and sky spatials rightward ...");
         } else {
             return;
         }
         /*
          * Update the local rotations of both objects based on the angle.
          */
-        System.out.printf(" to %.1f radians left of start%n", angle);
```

Comment:
I don't understand why adding second carriage return to a console message is considered a "performance improvement".

Reply:
I think this is old code? Yeah, better to SQUASH when merge

Reply:
These are related to pspeed's comments. So in few places this goes beyond performance improvements and actually changes the behavior to allegedly better one.

Reply:
In this case, looking at the other methods... they were meant to be prints to get them all on one line.

Then you added a _second_ carriage return to line 141.

My comments about line feeds were apparently incorrect.

Reply:
I accidentally added the second one yes, but I reverted it quite quickly afterwards (https://github.com/jMonkeyEngine/jmonkeyengine/pull/2220/commits/47a9e9596e4bced184b4b8de731b195f42d193fe). Somehow Stephen managed to review 4h old revision.

I'll revert the comment change then too.

Reply:
> I'll revert the comment change then too.

Done